README.md

```diff
@@ -61,9 +61,9 @@ npm install
 npm run start
 ```
 2. a browser window should automatically open at http://localhost:9000, if for some reason it doesn't go ahead and manually navigate there.
-3. edit `dev/pane/index.ts` and change the color string constant at the end of line 61.
-4. wait about 5 seconds
-5. you'll see the color of the box and heading change in your browser
+3. Once you arrive at the Solid Pane Tester page, the profile-pane will be loaded by default, Proceed to edit solid-panes/dev/loader.ts and at line 5, you should see import Pane from 'profile-pane'. Simply change 'profile-pane' to your preferred pane/directory containing the pane of choice, for example, you could choose 'source-pane' and bam, it will load that pane. For those who are new, you can go to the solid-panes/src directory and manually navigate through each individual folder. In most folders, you simply look for any file that has Pane in the title. Copy and paste the pane.js file of your choice into the solid-panes/dev/pane folder or you can simply import directly from the src directory. For example import from '../src/dokieli/dokieliPane' will work just fine. Each time you save solid-panes/dev/loader.ts while importing a different pane, your browser at http://localhost:9000/ should automatically refresh. It's a good idea to keep the developer console open in your web browser to ensure panes are loading and rendering properly. 
```

Comment:
```suggestion
3. Once you arrive at the Solid Pane Tester page, the `profile-pane` will be loaded by default. Proceed to edit `solid-panes/dev/loader.ts` and, at line 5, you should see `import Pane from 'profile-pane'`. Simply change `'profile-pane'` to your preferred pane/directory containing the pane of choice; for example, you could choose `'source-pane'` and bam, it will load that pane. For those who are new, you can go to the `solid-panes/src` directory and manually navigate through each individual folder. In most folders, you simply look for any file that has `pane` in the title. Copy and paste the `pane.js` file of your choice into the `solid-panes/dev/pane` folder, or you can import directly from the `src` directory. For example, importing from `'../src/dokieli/dokieliPane'` will work just fine. Each time you save `solid-panes/dev/loader.ts` while importing a different pane, your browser at `http://localhost:9000/` should automatically refresh. It's a good idea to keep the developer console open in your web browser to ensure panes are loading and rendering properly. 
```